### PR TITLE
Adds basic nhs config to sample config

### PR DESCRIPTION
**Pull request by whpearson**
So that there is less to add to get things up and running.

This sample config is copied to config.py during the setup of the local development environment.

### CODE REVIEW

**Review by philandstuff — COMMENTED** [review]
I don't understand this so I don't feel competent to ✅ . Can you talk me through it?

File: instance/config.py.sample.dev [hunk 1]
```diff
@@ -15,11 +15,12 @@ POSTCODE_TIER_OVERRIDE = """{"TE22RR" : {"tier": 2, "shielding": 0},
 # NHS OIDC config
 NHS_OIDC_AUTHORITY_URL = "https://auth.sandpit.signin.nhs.uk"
 NHS_OIDC_CLIENT_ID = "temp"
-NHS_OIDC_REGISTRATION_CALLBACK_URL = "http://localhost:5000/nhs-registration-callback"
-NHS_OIDC_LOGIN_CALLBACK_URL = "http://localhost:5000/nhs-login-callback"
-NHS_OIDC_LOGIN_PRIVATE_KEY_PATH = "instance/private_key.pem"
+NHS_OIDC_REGISTRATION_CALLBACK_URL = "http://webform:5000/nhs-registration-callback"
+NHS_OIDC_LOGIN_CALLBACK_URL = "http://webform:5000/nhs-login-callback"
```
Comment (whpearson) [inline instance/config.py.sample.dev:19]: The callback urls have changed to work in a docker-compose environment.

Comment (whpearson) [inline instance/config.py.sample.dev:19]: The callback urls have changed to work in a docker-compose environment.

File: instance/config.py.sample.dev [hunk 2]
```diff
@@ -15,11 +15,12 @@ POSTCODE_TIER_OVERRIDE = """{"TE22RR" : {"tier": 2, "shielding": 0},
...
+NHS_OIDC_SCOPES = ["openid", "profile", "email", "phone", "profile_extended"]
+NHS_OIDC_VTR = ["P0.Cp.Cd", "P0.Cp.Ck", "P0.Cm"]
```
Comment (whpearson) [inline instance/config.py.sample.dev:21]: The NHS_OIDC values are talked about here https://digital.nhs.uk/services/nhs-care-identity-service-2/guidance-for-developers/openid-connect-overview

The scope is the data we expect back from the NHS. The VTR is the vector of trust. https://nhsconnect.github.io/nhslogin/vectors-of-trust/ P0 is proof of email and phone number.

File: instance/config.py.sample.dev [hunk 3]
```diff
@@ -15,11 +15,12 @@ POSTCODE_TIER_OVERRIDE = """{"TE22RR" : {"tier": 2, "shielding": 0},
...
 NHS_OIDC_LOGIN_PRIVATE_KEY = ""
-
+NHS_OIDC_LOGIN_PRIVATE_KEY_PATH = "instance/private_key.pem"
```
Comment (whpearson) [inline instance/config.py.sample.dev:23]: This is an unintentional move.

**Comment by whpearson** [comment]
The context for this is trying to reduce the amount of documentation needed for this PR https://github.com/alphagov/covid-engineering/pull/1603/files

**Comment by whpearson** [comment]
I had a bunch of config in my config.py that was not actually needed. Amended the  PR to only the things needed and linked to the PR for why it is needed.

**Review by philandstuff — APPROVED** [review]

**Pull request merged**